homeassistant/components/traccar/device_tracker.py

```diff
@@ -30,6 +31,25 @@
 ATTR_MOTION = 'motion'
 ATTR_SPEED = 'speed'
 ATTR_TRACKER = 'tracker'
+ATTR_TRACCAR_ID = 'traccar_id'
+
+EVENT_DEVICE_MOVING = 'deviceMoving'
```

Comment:
We usually have config options be lowercase snakecase.

Reply:
Are you referring to ATTR_TRACCAR_ID?

Reply:
Those are the events as defined in the traccar API, if I convert them to lowercase snakecase I would need to translate them to the traccar defined ones when calling the API. Is this ok?

Reply:
Yes, it's fine to translate them.